scripts/workers/code_upload_submission_worker.py

```diff
@@ -567,6 +589,19 @@ def update_failed_jobs_and_send_logs(
                             logger.exception(
                                 "Exception while reading Job logs {}".format(e)
                             )
+                    elif is_submission_container_executed(
```

Comment:
@savish28 Check if it's a submission container as a first condition here itself. You can remove it from inside the function

Reply:
@KhalidRmb We were checking the conditions here only but for better readability, we shifted the checks to the function. Do we want the submission condition to be segregated from the function?

Reply:
Hmm...just the one check. Because why enter the function if it's not the submission container

Reply:
Updated